digiroad2-viite/src/test/scala/fi/liikennevirasto/viite/process/TrackSectionOrderSpec.scala

```diff
@@ -297,4 +313,32 @@ class TrackSectionOrderSpec extends FunSuite with Matchers {
     chainEndPointLink._1 should be (chainEndPointLink._2.endPoint)
   }
 
+  test("Test orderProjectLinksTopologyByGeometry When a part of a road is transfered to another then road sidecodes should be equal and the transfered part ith opposite sidecode should be reversed.") {
+    val points1      = Seq(Point(0, 0), Point(10, 0))
+    val points2      = Seq(Point(10, 0), Point(20, 0))
+    val points3      = Seq(Point(20, 0), Point(30, 0))
+    val (rwn1, rwn2) = (99998, 99999)
+
```

Comment:
Tähän sanallinen esittely datasta, sen ymmärryksen helpottamiseksi, mitä yritetään saavuttaa.
"Two links on an unchanging roadway (0), and third that is on a changing roadway (1) to be reversed, and side code to be changed at ordering."
tms.